planner/core/casetest/testdata/integration_partition_suite_out.json

```diff
@@ -615,7 +615,7 @@
         "SQL": "explain format = 'brief' insert into tcollist select * from tcollist where a in (2)",
         "DynamicPlan": [
           "Insert N/A root  N/A",
-          "└─TableReader 10.00 root partition:p0 data:Selection",
+          "└─TableReader 10.00 root partition:p0,p1,p2,p3 data:Selection",
```

Comment:
Weird... It seems like the partition pruning cannot work with `list column` partitioning tables.